terraform/ggroup-to-splunk/lambda_iam.tf

```diff
@@ -4,10 +4,10 @@ data "template_file" "lambda_trust" {
 }
 
 data "template_file" "lambda_policy" {
-  template = file("${path.module}/json/interface_lambda/policy.json")
+  template = file("${path.module}/lambda_json/policy.json")
```

Comment:
Can you use a (policy document)[https://www.terraform.io/docs/providers/aws/d/iam_policy_document.html] instead of the json template.

Reply:
Done